src/Sentry/Integrations/WinUIUnhandledExceptionIntegration.cs

```diff
@@ -73,6 +75,11 @@ public void Register(IHub hub, SentryOptions options)
         });
     }
 
+#if !TRIMMABLE
+    /// <summary>
```

Comment:
This might have been fixed since this hack was done. @lucas-zimerman might know if there's a non-reflection way around this.

Reply:
I think we'll need to document what features are not supported in an AOT scenario then:

* WinUI apps won't have unhandled exceptions?

Reply:
OK just realized this is on the PR desc, so it'll be easier to write on the CHANGELOG and later on the docs